default.nix

```diff
@@ -0,0 +1,13 @@
+# Desc: Default.nix with flake-compat
```

Comment:
This too, is a helper file for nixOs packaging, which I'll do soon (so this app will be in the official NixOs repo `nixpkgs`)